pb_app/lib/submission_form.dart

```diff
@@ -24,80 +25,90 @@ class _SubmissionFormScreenState extends State<SubmissionFormScreen> {
                 ),
                 Expanded(
                   child: AnimatedBuilder(
                     animation: _nameFocusNode,
                     builder: (context, _) {
-                      return TextField(
-                        autofocus: true,
-                        focusNode: _nameFocusNode,
-                        textAlign: TextAlign.center,
-                        style: const TextStyle(fontSize: 20),
-                        decoration: InputDecoration(
-                          border: InputBorder.none,
-                          hintText: _nameFocusNode.hasFocus ? '' : 'Name',
+                      return Container(
+                        decoration: BoxDecoration(
+                          borderRadius: platformAwareBorderRadius(10),
+                          color: Colors.black12,
+                        ),
+                        child: TextField(
+                          autofocus: true,
+                          focusNode: _nameFocusNode,
+                          textAlign: TextAlign.center,
+                          style: const TextStyle(fontSize: 20),
+                          cursorColor: Colors.black26,
+                          decoration: InputDecoration(
+                            border: InputBorder.none,
+                            hintText: _nameFocusNode.hasFocus ? '' : 'Name',
+                          ),
                         ),
                       );
                     },
                   ),
                 ),
-
-                /// Dummy IconButton to center the text field
-                ///
-                /// Learn the rules like a scientist
-                /// so you can break them like an artist.
                 const Visibility(
                   visible: false,
                   maintainSize: true,
                   maintainAnimation: true,
                   maintainState: true,
                   child: IconButton(
-                    icon: Icon(Icons.close),
+                    icon: Icon(Icons.close_rounded),
                     onPressed: null,
                   ),
                 ),
               ],
             ),
           ),
         ),
-      ),
-      body: PageView(
-        children: const [
-          _Card('Home Screen'),
-          _Card('Lock Screen'),
-        ],
-      ),
-      bottomNavigationBar: SafeArea(
-        minimum: const EdgeInsets.symmetric(horizontal: 25, vertical: 15),
-        child: ElevatedButton(
-          onPressed: () {},
-          style: ElevatedButton.styleFrom(
-            shape: SmoothRectangleBorder(
-              borderRadius: platformAwareBorderRadius(10),
+        body: PageView.builder(
+          controller: _pageController,
+          itemCount: 2,
+          physics: const BouncingScrollPhysics(),
+          itemBuilder: (context, index) => index == 0
+              ? const _Card('Home Screen')
+              : const _Card('Lock Screen'),
```

Comment:
```suggestion
         body: PageView(
           children: const [
            _Card('Home Screen'),
            _Card('Lock Screen'),
          ],
```

We won't see a performance benefit using PageView.builder here, so the added complexity is not worth it